i18n/states.php

```diff
@@ -172,6 +172,85 @@
 	),
 	'BH' => array(),
 	'BI' => array(),
+	'BJ' => array( // Benin states.
+		'BJ-01' => __( 'ABOMEY', 'woocommerce' ),
```

Comment:
Those state names doesn't seems to match the list: https://github.com/unicode-org/cldr/blob/master/common/subdivisions/en.xml#L522-L533

Also it's all on uppercase, it need to be fixed.

Reply:
Okay i'll apply the changes and commit it

Reply:
I have applied changes as mentionned, does it match now?

Reply:
@claudiosanches